pages/dashboard/mentor/calendar.page.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Sidebar from "../../../components/mentorDashboard/sidebar";
+import Calendar from "../../../components/mentorDashboard/calendar";
+
+function calendar() {
+  return (
+    <>
+      <Sidebar />
+      <Calendar />
+    </>
+  );
+}
+
+export default calendar;
```

Comment:
Oh i was thinking to render components on the basis on hash in url....like
url m `currentTab = profile`  h toh profile component render ho..
I mean aise kaafi page.js banane pdenge.

Reply:
> I mean aise kaafi page.js banane pdenge.

I agree, so okay we will go with your approach